Gemfile.lock

```diff
@@ -340,10 +340,10 @@ GEM
       rails-assets-angular (= 1.4.7)
     rails-assets-angular-scenario (1.4.7)
       rails-assets-angular (= 1.4.7)
-    rails-assets-angular-ui-bootstrap-bower (0.14.3)
-      rails-assets-angular (>= 1.3.0)
     rails-assets-angular-truncate-2 (0.4.1)
       rails-assets-angular (>= 1.2.0, < 2)
+    rails-assets-angular-ui-bootstrap-bower (0.14.3)
+      rails-assets-angular (>= 1.3.0)
```

Comment:
Not sure why existing Gemfile.lock had them ordered wrong. They should have been alphabetized already.